jax/_src/compilation_cache.py

```diff
@@ -219,16 +232,19 @@ def put_executable_and_time(
   min_entry_size = config.persistent_cache_min_entry_size_bytes.value
   entry_size = len(executable_and_time)
   if entry_size < min_entry_size:
-    logger.info(
-        "Not writing cache entry with key %s since its size (%d bytes) "
-        "is less than threshold (%d bytes)",
+    logger.log(
+        log_priority,
+        "Not writing persistent cache entry with key %s since its size"
+        " (%d bytes) is less than threshold (%d bytes)",
         cache_key,
         entry_size,
         min_entry_size,
     )
   else:
-    logger.info(
-        "Writing %s to persistent compilation cache with key %s.",
+    logger.log(
+        log_priority,
+        "Writing %s to persistent compilation cache with key %s."
+        " Entry does not exist yet.",
```

Comment:
nit: is the "Entry does not exist yet" necessary? I feel like that's expected if it's writing to the cache